docs/reference/nodes.md

```diff
@@ -1,5 +1,68 @@
+
+
+![Node UI](../../images/node_ui.png "Node UI")  
+
+* **Node Name**: Defines the name of the node. On creation, the name of the node is the name of the operator with an enumerated suffix. The name of the node can be changed through the [*Inspector Widget*](../getting_started/basics/ui_overview/#inspector).
+* **Operator Type**: The associated operator to the node. It determines the native plugs of the node.
+* **Trigger Plugs**: The trigger plugs (`>>>`) are plugs meant to define the node's position in the execution graph. They do not pass any data. If the node has other input or output plugs connected to another node, the trigger plugs do not need to be connected; otherwise, it is recommended to connect these plugs to indicate when the node needs to get computed.
+
+* **I/O's Plugs**: Input and output plugs are define the incoming data required by the operator and the resulting data after it is computed. 
+
+### Inspect a Node
+To read further information about a node, `Right-Click` on the node and `Click` on the *Help* option. This will prompt a dialog with the following information:
+
+* The node's name.
+* The operator type and the [Catalog](../reference/catalogs/#catalogs) it belongs to.
+* The operator description.
+* The input and output plugs with their corresponding plug types.
+
+![Shift Node Information Dialog](../images/shift_operator_dialog.gif)
+
+## Plugs
+Plugs define the input and output data of the nodes. Data values can be set through the [*Inspector Widget*](../getting_started/basics/ui_overview/#inspector) or by connecting plugs, which allows nodes to pass data from one to another. The kind of data compatible with a plug is defined by the plug type.
+
+**Plug Types**
+
+| Type | Description | Compatible Python Type |
+| :--- | :---------- | :---|
+| *Bool* | Defines a boolean value.|`Bool`|
+| *Code* | Defines a python code block. This type is used whenever custom scripting is required in a node. A plug of this type displays as an embedded text editor in the inspector. |`String`|
+| *Color* | Defines a color in RGBA. Interacting with a plug of this type opens a color picker dialog.|`List`|
+| *Dict* | Defines a dictionary value.|`Dict`|
+
+| *Dir* | Defines a directory path. Interacting with a plug of this type opens a directory browser dialog.|`String`|
+| *FileIn* | Defines an input file. This type is used for open and import operations. Interacting with a plug of this type opens a file browser dialog.|`String`|
+| *FileOut* | Defines an output file. This type is used for save and export operations. Interacting with a plug of this type opens a file browser dialog.|`String`|
+| *Float* | Defines a float value.|`Float`|
+| *Instance* | Defines an object instance. These plugs can be differentiated by their dotted outline.|`Any`|
+| *Int* | Defines a integer value.|`Int`|
+| *List* | Defines a list value.|`List`|
+| *Object* | Defines a generic Python object.|`Any`|
+| *String* | Defines a string value.|`String`|

```

Comment:
I think we are missing Enumerator plugs and the triggers

Reply:
True! I only listed the ones that can be created. I think Enumerator plugs make sense to add it since the user interacts with them, however, trigger plugs are explained when going over the Node interface.
Do you think it should be mentioned again here?

Reply:
I added the Enumerator type

Reply:
Well, about triggers, I´m not sure. I would say it´s better add them, so this table can be consider like a reference of all plug type, not only custom user plugs, or plug to be created in a operator. But we can ask to the team if you prefer. Maybe you think is better to skip them.

Reply:
Done